data/gen/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ipld/go-ipld-prime/schema"
+	gengo "github.com/ipld/go-ipld-prime/schema/gen/go"
+)
+
+func main() {
```

Comment:
make sure to add a `go:generate` line somewhere in this repo so that a `go generate ./...` at the root re-runs this.